### PR TITLE
Update powered-by_adding SPITHA.html

### PR DESCRIPTION
**Pull request by VictorParkM**
merging latest asf-site  
This is a modification to the existing request based on the advise from Matthias : adding spitha on powered-by #515

### CODE REVIEW

File: powered-by.html [hunk 1]
```diff
@@ -801,6 +801,11 @@
         "logo": "edenlab.svg",
         "logoBgColor": "#ffffff",
         "description": "Edenlab FHIR engineers use Kafka to manage real-time data flow between dozens of microservices in our product - <a href=https://kodjin.com>Kodjin Interoperability Suite</a> - a low-code FHIR-based infrastructure for healthcare data management (contains FHIR Server, Terminology Service, and Mapper). Edenlab is a custom software and product development company focused primarily on healthcare data interoperability. We are also working with some major customers from the Fintech space (like Mastercard ® and Raffeisen BankInternational)."
+    }, {
+	"link': "https://spitha.io/",
+	"logo": "spitha.png",
```
Comment (mjsax) [inline powered-by.html:806]: The PR is missing the corresponding png file.

Comment (VictorParkM) [reply powered-by.html:806]: I added the png file

**Comment by mjsax** [comment]
@VictorParkM -- thanks for updating your webpage adding `®` to Apache Kafka. However, you should also have a disclaimer on the web page about it.

>  Apache®, Apache Kafka®, Kafka®, and associated open source project names are trademarks of the [Apache Software Foundation](https://www.apache.org/)

Cf [confluent.io :](https://www.confluent.io/) who has a similar thing at the bottom of their landing page.

**Comment by VictorParkM** [comment]
Hi Matthias,

I fixed all the flaws regarding ASF guidelines.


Regards,

Victor Park

JeongHun Park

Business Development / Team Manager

SPITHA INC.
43, Daesagwan-ro, Yongsan-gu, 04401
SEOUL, SOUTH KOREA
M) +82.010.4321.2320
E) ***@***.*** ***@***.***>

*spitha.io <http://spitha.io/>*



The content of this email is confidential and intended for the recipient
specified in message only. It is strictly forbidden to share any part of
this message with any third party, without a written consent of the sender.
If you received this message by mistake, please reply to this message and
follow with its deletion, so that we can ensure such a mistake does not
occur in the future.


2024년 7월 9일 (화) 오전 9:55, Matthias J. Sax ***@***.***>님이 작성:

**Comment by mjsax** [comment]
Thanks for getting back @VictorParkM -- I did look at the webpage but cannot see any changes? The `(R)` signs are still there, but I cannot find a "copyright disclaimer" -- Did I miss it? Can you point me to it?

**Comment by VictorParkM** [comment]
You can find the copyright disclaimer at the bottom of the page.
[image: image.png]
Thank you




JeongHun Park

Business Development / Team Manager

SPITHA INC.
43, Daesagwan-ro, Yongsan-gu, 04401
SEOUL, SOUTH KOREA
M) +82.010.4321.2320
E) ***@***.*** ***@***.***>

*spitha.io <http://spitha.io/>*



The content of this email is confidential and intended for the recipient
specified in message only. It is strictly forbidden to share any part of
this message with any third party, without a written consent of the sender.
If you received this message by mistake, please reply to this message and
follow with its deletion, so that we can ensure such a mistake does not
occur in the future.


2024년 7월 18일 (목) 오후 12:14, Matthias J. Sax ***@***.***>님이 작성:

**Comment by mjsax** [comment]
Ah thanks. It's there now. Did not see it yesterday. Not sure why...

**Pull request merged**